packages/techdocs-cli/src/lib/utility.ts

```diff
@@ -44,7 +44,7 @@ export const createLogger = ({
   verbose: boolean;
 }): winston.Logger => {
   const logger = winston.createLogger({
-    level: verbose ? "verbose" : "info",
+    level: verbose ? "debug" : "info",
```

Comment:
should this change be reverted back or the verbose param could be changed?

Reply:
Yeah, perhaps we should revert this back to `verbose` and update the code in `techdocs-common` that logs out each migrated file to log at `verbose` level rather than `debug` level?